deployments/linea-goerli/usdc/configuration.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xf56dc6695cF1f5c364eDEbC7Dc7077ac9B586068",
+  "baseTokenPriceFeed": "0xcb81fffa91cd5f3436318d25057c1d5df8e23f95",
+  "borrowMin": "100e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "1000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000011574074074074073e15",
+    "baseBorrowSpeed": "0.0011458333333333333e15",
```

Comment:
Ideally, we set these as 0 on deploy and increase them with a cross-chain governance proposal.